Week1/homework/app.js

```diff
@@ -1,11 +1,53 @@
 'use strict';
-{
+function load() {
+  const root = document.querySelector("#books");
+
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    "suhuf",
+    "tawrat",
+    "zabur",
+    "injeel",
+    "quran"
   ];
+  const bookInfo = {
+    "suhuf": { title: "Scrolls of Abraham", auther: "Ibrahim", lang: "Unknown" },
+    "tawrat": { title: "Torah of Moses", auther: "Musa", lang: "Unknown" },
+    "zabur": { title: "Psalms of David", auther: "Dawud", lang: "Unknown" },
+    "injeel": { title: "Gospel of Jesus", auther: "Isa", lang: "Syriac" },
+    "quran": { title: "Quran", auther: "Muhammad", lang: "Arabic" }
+  };
+
+  const covers = {
+    suhuf: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT0BSSvkQ8zw7-yJOa1a1j8yhb3jzLFMSCdNLHoNQCfP3Zl0BVSyg",
+    tawrat: "https://www.thoughtco.com/thmb/uxVKjHrNQuinx_V1AD60jBkqhAk=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/BookMoses-56a536713df78cf77286f661.jpg",
+    zabur: "https://www.thoughtco.com/thmb/tJaImQdvAb1oCthBxC96CkSDSNw=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/Psalms-56a536713df78cf77286f65e.jpg",
+    injeel: "https://www.thoughtco.com/thmb/vnpJMnQ9LoEdJYOdbcZDs5ILwwE=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/Gospel-56a536715f9b58b7d0db88ae.jpg",
+    quran: "https://www.thoughtco.com/thmb/KqhAnEZ9daNpd3D29OxyM6RFZj4=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/quran-56a5365c3df78cf77286f5a6.jpg"
+  };
 
+  function createEl(name, parent, nodes = {}) {
+    const elem = document.createElement(name);
+    parent.appendChild(elem);
+    if (nodes.txt) { elem.innerHTML = nodes.txt; }
+    if (nodes.id) {
+      elem.setAttribute(nodes.id, nodes.val);
+    }
+    return elem;
+  }
 
-  // Replace with your own code
-  console.log(bookTitles);
+  const ul = createEl("ul", root);
+  for (let i of bookTitles) {
```

Comment:
If you are looping through `bookTitles`, a single element of that array would be a `bookTitle` (or simply `title`). So that should be the name for the loop variable, instead of `i`.